internal/commands/flags.go

```diff
@@ -47,11 +47,16 @@ var (
 	repositoryUrlFlagName  = "repository-url"
 	repositoryUrlShortFlag = "r"
 
-	// repositoryUrl the path to the repository to scan
+	// accessToken the access token to use for the repository
 	accessToken          string
 	accessTokenFlagName  = "access-token"
 	accessTokenShortFlag = "t"
 
+	// scm hosting the repository
+	scm          string
```

Comment:
I believe it more accurate to call it self-hosted-server

Reply:
my bad, what about `self-hosted-scm`.  I would like to emphasize that the parameter is required only for on-prem scenario

Reply:
It is not entirely about self-hosted (despite my initial PR title). I was reminded that both GH and GL are starting to offer dedicated SaaS environments on custom domains.

To keep things moving, I've no objection to changing to `self-hosted-scm` since it will likely be most relevant to current deployment models. Although worth noting it might change.

Reply:
I've updated with `scm-platform` to test out an alternative.